src/pages/dongheeKim/Login/Login.js

```diff
@@ -1,7 +1,116 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import './Login.scss';
 
-function Login() {
-  return <div>Nav bar</div>;
+class LoginDonghee extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      first_name: 'Donghee',
+      last_name: 'Kim',
+      email: '',
+      password: '',
+      phone_number: '01011111111',
+      gender: 'w',
+      birth: '1995-12-27',
+    };
+  }
+
+  /*--회원가입--*/
+  handleJoin = () => {
+    fetch('http://10.58.7.144:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        first_name: 'Donghee',
+        last_name: 'Kim',
+        email: this.state.email,
+        password: this.state.password,
+        phone_number: '010-1111-1111',
+        gender: 'w',
+        birth: '1995-12-27',
+      }),
+    })
+      .then(response => response.json())
+      .then(response => {
+        if (response.MESSAGE === 'SUCCESS') {
+          alert('회원가입 성공!');
```

Comment:
success 면 alert 출력하고 함수를 끝내버리는게 좋을 것 같네요 안그러면 함수 본문 전체를 다 읽고나서 함수가 종료됩니다.
```suggestion
          return alert('회원가입 성공!');
```

Reply:
아하~ 본문을 다 읽게되면 효율성이 떨어지겠군요! 제대로 구현하는 것만 확인하다보니 효율성까지 제대로 고려하지는 못한 것 같습니다. 반영했습니다! 😎